content/learn/06.hardware/nano-pcb-guide/nano-pcb-guide.md

```diff
@@ -0,0 +1,158 @@
+| Humidity    | x    | x          | x           | **HTS221**        | x              | x                   |
+
+***\*Earlier versions of the Nano 33 IoT uses the LSM6DS3 module, which can be accessed via the [LSM6DS3 library](https://www.arduino.cc/reference/en/libraries/arduino_lsm6ds3/). For newer versions using the LSM6DSOX module, use the [LSM6DSOX library](https://www.arduino.cc/reference/en/libraries/arduino_lsm6dsox/).***
+
+## Power Considerations
+
+### Voltage (3.3V / 5V)
+
+It is important to understand that the Nano family boards operates on different voltage. Any board with a radio module (Nano 33 BLE, Nano 33 BLE Sense, Nano 33 IoT, Nano RP2040 Connect) operates on **3.3V**. The Nano and Nano Every operates on **5V**.
+
+As the boards with radio modules operate on 3.3V logic, the 5V pin is connected to headers via a solder jumper which defaults open. When powering the board via USB, the VIN pin can be used as a 5V output from the board. This is useful when powering carrier boards requiring 5V and powering the system via USB.
+
+### VUSB Pads
+
+The 5V pin is also referred to as **"VUSB"**. The purpose of this pin is to provide direct connection between USB connector's VUSB (5V) to headers. This can be used to power devices on a carrier board directly while when using VIN pin this will exhibit a drop due to the series diodes that prevents VIN to back power the USB. In order to connect VUSB/5V pin headers, you will need to solder together the VUSB pads on the bottom of the board, as shown in the image below:
+
+![Solder the VUSB pads.](assets/5V-PIN-VUSB.png)
+
+***If you solder the VUSB pads, and then power the board via USB, it also enables the 5V pin to be used. Be very cautios with this, as you risk damaging your board's ICs.***
+
+### VIN Min/Max
+
+The min/max voltage supply varies between boards. This is important to consider when choosing the battery source, that you do not exceed the limits to damage the board. 
+
+| Nano  | Nano Every | Nano 33 BLE | Nano 33 BLE Sense | Nano 33 IoT | Nano RP2040 Connect |
+| ----- | ---------- | ----------- | ----------------- | ----------- | ------------------- |
+| 7-12V | 7-18V      | 5-18V       | 5-18V             | 5-18V       | 5-18V               |
+
+### Battery Connection
+
+The Nano family was not designed to be battery powered, so it has no battery charging circuit, and does not have battery under voltage protections.
+
+If you want to connect a battery to a Nano, do so by using the VIN pin but you have to keep in mind VIN minimum requirements (which means a single cell battery won't be enough). Since there's no under voltage protection, a battery without a protection circuit will be damaged if it is kept connected to the Nano board even after it is completely discharged. It is thus recommended not to connect a battery directly but rather to use circuits like those in USB pattery packs that provide a stabilized 5V power supply and protect the battery from undervoltage.
```

Comment:
```suggestion
If you want to connect a battery to a Nano, do so by using the VIN pin but you have to keep in mind VIN minimum requirements (which means a single cell battery won't be enough). Since there's no under voltage protection, a LiPo battery without a protection circuit will be damaged if it is kept connected to the Nano board even after it is completely discharged. It is thus recommended not to connect a battery directly but rather to use circuits like those in USB battery packs that provide a stabilized 5V power supply and protect the battery from undervoltage.
```

- Specify battery type which is damaged by over discharge
- Fix typo

I also am skeptical about whether this warning about over discharge is necessary. I have never encountered a consumer LiPo battery without an integrated BMS. The people using batteries without one will be aware of the fact and the implications.

Reply:
A large number of them would have built in circuits to protect it, but not all. @pnndra what do you say?